src/libraries/System.Text.Json/tests/TrimmingTests/SerializerEntryPoint/Deserialize.FromReader.BoxedObject.cs

```diff
@@ -20,15 +20,15 @@ static int Main(string[] args)
             string json = "[1]";
             Utf8JsonReader reader = new Utf8JsonReader(Encoding.UTF8.GetBytes(json));
             int[] arr = (int[])JsonSerializer.Deserialize(ref reader, typeof(int[]));
-            if (!TestHelper.VerifyWithSerialize(arr, json))
+            if (arr == null)
```

Comment:
I think we should verify the object is correct.
```suggestion
            if (arr == null || arr.Length != 1 || arr[0] != 1)
```